compactor/src/compact.rs

```diff
@@ -779,7 +785,7 @@ pub mod tests {
         // Create a compactor
         let time_provider = Arc::new(SystemProvider::new());
         let mut config = make_compactor_config();
-        // 8 hours to be cold
+        // 8 hours to be cold (this is the default)
```

Comment:
For tests, I created a config with time to be cold to 10 minutes. See here https://github.com/influxdata/influxdb_iox/blob/654ed98d1f5bb77243417e1e5de0738d5bd44d98/compactor/src/compact.rs#L692

In this test, I make it 8 hours so all behaviors in the tests are the same as it was with previous default 8 hours. I think it is appropriate not to mention the default here. We just test with 8 hours
```suggestion
        // 8 hours to be cold
```